src/tiles/grid/item-tile.ts

```diff
@@ -109,6 +120,19 @@ export class ItemTile extends LitElement {
     `;
   }
 
+  private get textSnippetsTemplate(): TemplateResult | typeof nothing {
+    if (!this.hasSnippets) return nothing;
+
+    return html`<text-snippet-block
+      viewsize="grid"
+      .snippets=${this.model?.snippets}
+    ></text-snippet-block>`;
+  }
+
+  private get hasSnippets(): boolean {
+    return !!(this.model?.snippets && this.model.snippets.length > 0);
```

Comment:
can we do `this.model?.snippets?.length` ?

Reply:
👍